framework/OpenMod.Core/Permissions/DefaultPermissionRoleStore.cs

```diff
@@ -71,6 +73,8 @@ public virtual async Task<IReadOnlyCollection<IPermissionRole>> GetRolesAsync(IP
             // ReSharper disable once ForeachCanBePartlyConvertedToQueryUsingAnotherGetEnumerator
             foreach (var roleId in userData.Roles)
             {
+                if (string.IsNullOrEmpty(roleId)) continue;
```

Comment:
For future reference: please no one-liner ifs, instead use brackets 
```cs
if (string.IsNullOrEmpty(roleId)
{
    continue;
}
```

Reply:
Sounds good